app/components/nextflow_component.html.erb

```diff
@@ -25,7 +25,7 @@
     <%= @workflow.description %>
   </p>
   <%= form_with url: url, data: { turbo_frame: "_top" } do |form| %>
-    <input type="hidden" name="format" value="turbo_stream"/>
+    <input id="submision_turbo" type="hidden" name="format" value="turbo_stream" />
```

Comment:
Typo in `submision_turbo`

Reply:
Fixed in 22aed8d8f147a9891b93be8b9846414a59874cb2